csrc/cpu/adam/simd.h

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0
+
+// DeepSpeed Team
+
+#pragma once
```

Comment:
This is also similar to `csrc/includes/simd.h`. It will get good to merge to avoid duplicate files in the repo.

Reply:
This file is identical to `csrc/includes/simd.h`, let me see if the duplication could be removed.